src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -473,6 +496,39 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 	return nil
 }
 
+func (c controller) getReadyOperators() ([]string, []models.MonitoredOperator, error) {
```

Comment:
Maybe I'm wrong, but it seems like we are waiting twice for CSVs to be available.
It should wait for them to be created, and only after everything else finished, we need to look for them to be available (as it is implement ATM)

Reply:
The problem is that, the CSV report the state the same way `status=succeed` in case of the subscription and CRs, so that's why we wait for same condition. But it's different.